File: Client/src/components/AdminPage/Player.js
import React from 'react'

export default function Player(props) {
    const {name, lastName, age, number, position} = props;
  return (
    <div style={{display:"flex", 
                 justifyContent:"center", 
                 alignItems:"center",
                 flexDirection:"column",
                 backgroundColor:"rgba(120, 120, 120, 0.6)",
                 border:"2px solid white"}}>
        <h3 style={{color:"white"}}>Player to be added</h3>
        <p style={{color:"white"}}>Name: {name}</p>
        <p style={{color:"white"}}>lastName: {lastName}</p>
        <p style={{color:"white"}}>Age: {age}</p>
        <p style={{color:"white"}}>Number: {number}</p>
        <p style={{color:"white"}}>Position: {position}</p>
    </div>
  )
}
// Player.defaultProps = {
//     name : "",
//     lastName : "",
//     age: 0,
//     position: ""
// }

File: Backend/SpartanManageFootball/SpartanManageFootball/Models/Player.cs
using System.ComponentModel.DataAnnotations;

namespace SpartanManageFootball.Models
{
    public class Player
    {
        [Key]
        public int Id { get; set; }
        [Required(ErrorMessage = "Name is required")]
        public string Name { get; set; }

        [Required(ErrorMessage = "Last name is required")]
        public string LastName { get; set; }

        [Required(ErrorMessage = "Age is required")]
        public int Age { get; set; }

        [Required(ErrorMessage = "Number is required")]
        public int Number { get; set; }

        [Required(ErrorMessage = "Position is required")]
        public string Position { get; set; }
        public int SquadTeamId { get; set; }
    }
}
